modules/apps/knowledge-base/knowledge-base-web/src/main/java/com/liferay/knowledge/base/web/internal/display/context/KBTemplatesManagementToolbarDisplayContext.java

```diff
@@ -126,6 +126,30 @@ public CreationMenu getCreationMenu() {
 		).build();
 	}
 
+	public List<DropdownItem> getEmptyStateActionDropdownItems() {
+		return DropdownItemListBuilder.add(
+			() ->
+				Validator.isNull(_getKeywords()) &&
+				AdminPermission.contains(
+					_themeDisplay.getPermissionChecker(),
+					_themeDisplay.getScopeGroupId(),
+					KBActionKeys.ADD_KB_TEMPLATE),
+			dropdownItem -> {
+				dropdownItem.setHref(
+					PortletURLBuilder.createRenderURL(
+						_liferayPortletResponse
+					).setMVCPath(
+						"/admin/common/edit_kb_template.jsp"
+					).setRedirect(
+						PortalUtil.getCurrentURL(_httpServletRequest)
+					).buildPortletURL());
+
```

Comment:
Small thing: you need to remove this blank line.

Reply:
done [48f0ad3](https://github.com/liferay-lima/liferay-portal/pull/3481/commits/48f0ad33f998128faee953b55d366399e3f22054)